internal/operators/manager.go

```diff
@@ -463,13 +449,24 @@ func EnsureLVMAndCNVDoNotClash(cluster *common.Cluster, openshiftVersion string,
 		}
 	}
 
-	if lvmEnabled && cnvEnabled {
-		return errors.Errorf("Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.")
+	if !lvmEnabled || !cnvEnabled {
+    return nil
+	}
+
+	// Openshift version greater or Equal to 4.12.0 support cnv and lvms
+  if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmsMinOpenshiftVersion4_12, openshiftVersion); !isGreaterOrEqual {
+		return errors.Errorf("Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.") 
 	}
 
-	if lvmEnabled && !common.IsSingleNodeCluster(cluster) {
+  if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmMinMultiNodeSupportVersion, openshiftVersion); isGreaterOrEqual {
```

Comment:
I think that this condition should be before 
// Openshift version greater or Equal to 4.12.0 support cnv and lvms

Reply:
yes I check for single mode after this if, on 4.15 we don't need to check if it `singleNodeCluster`